ceph-salt-formula/salt/ceph-salt/software.sls

```diff
@@ -27,3 +33,16 @@ upgrades disabled:
 {% endif %}
 
 {{ macros.end_stage('Install and update required packages') }}
+
+{% if pillar['ceph-salt'].get('upgrades', {'reboot': True})['reboot'] %}
+
+reboot:
+   ceph_salt.reboot_if_needed:
+     - failhard: True
+
+{% else %}
```

Comment:
You don't need the `else` block because the file will never be empty

Reply:
Done